src/main.rs

```diff
@@ -135,15 +165,7 @@ fn main() -> Result<()> {
         }
     }
 
-    // setup terminal
-    enable_raw_mode()?;
-    let mut stdout = io::stdout();
-    execute!(stdout, EnterAlternateScreen)?;
-    let backend = CrosstermBackend::new(stdout);
-    let mut terminal = Terminal::new(backend)?;
-    terminal.clear()?;
```

Comment:
Did you still need to do this? Wasn't sure why this was here anyway since you're switching to the alt buffer, so maybe it was just unnecessary?

Reply:
No, I didn't. I also removed the `clear()` from the Drop impl. It was incorrectly clearing the primary screen.